server/game/cards/13.5-TB/TyrionLannister.js

```diff
@@ -0,0 +1,35 @@
+const DrawCard = require('../../drawcard');
+const {Tokens} = require('../../Constants');
+
+class TyrionLannister extends DrawCard {
+    setupCardAbilities(ability) {
+        this.reaction({
+            when: {
+                onCardOutOfShadows: event => event.card.controller === this.controller 
+                                          && event.card.getType() !== 'event'
+            },
+            limit: ability.limit.perRound(1),
+            target: {
+                cardCondition: card => card.location === 'play area'
+                                    && card.controller === this.controller
+            },
+            handler: context => {
+                this.game.addMessage('{0} uses {1} to put a card into shadow', context.player, this);

```

Comment:
The message should indicate which card was put into shadows.

Reply:
fixed and thanks